conf.d/done.fish

```diff
@@ -175,7 +175,7 @@ if test -z "$SSH_CLIENT" # not over ssh
                 if test $exit_status -ne 0
                     set urgency "--urgency=critical"
```

Comment:
This also needs to be changed, otherwise the command will fail because the option name is already built in the variable.

Reply:
Done in the latest commit